### PR TITLE
 DDL.sql

### PR DESCRIPTION
**Pull request by SharifovDeveloper**

### CODE REVIEW

File: sql/DDL.sql [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+CREATE TABLE Review
+(
+    Id INT PRIMARY KEY IDENTITY NOT NULL,
+	Booking_Id INT NOT NULL,
+	Comment VARCHAR(500),
+	Grade  INT ,
+	FOREIGN KEY (Booking_Id) REFERENCES Booking(Id)
+)
+
```
Comment (diyorbekaxmatov) [inline sql/DDL.sql:89]: All  very Good MuhammadAli

**Comment by SharifovDeveloper** [comment]
Thank you

File: sql/DDL.sql [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+CREATE TABLE Booking
+(
+    Id INT PRIMARY KEY IDENTITY NOT NULL,
+    Guest_Id INT NOT NULL,
+	Employee_Id INT NOT NULL,
+	Room_Id INT NOT NULL,
+	Check_In_Date DATE,
+    Check_Out_Date DATE,
+	Total_Price MONEY
```
Comment (diyorbekaxmatov) [inline sql/DDL.sql:75]: Pay attention to the correctness of the lines

**Review by diyorbekaxmatov — CHANGES_REQUESTED** [review]

File: sql/DDL.sql (hunk 2 above)
Comment (SharifovDeveloper) [reply sql/DDL.sql:75]: Thank you I understanded

File: sql/DDL.sql [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+DROP TABLE  Document 
+DROP TABLE  Position
+
+
+----------------------------
+
+CREATE TABLE Document
+(
+    Id INT PRIMARY KEY IDENTITY NOT NULL,
+	Serial_Number VARCHAR(255),
```
Comment (diyorbekaxmatov) [inline sql/DDL.sql:17]: Again,this is a mistake

**Review by diyorbekaxmatov — APPROVED** [review]

File: sql/DDL.sql (hunk 3 above)
Comment (SharifovDeveloper) [reply sql/DDL.sql:17]: Oh Bro ,Sorry

File: sql/DDL.sql [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+DROP TABLE  Room  
+DROP TABLE  Category 
+DROP TABLE  Guest  
+DROP TABLE  Employee 
+DROP TABLE  Document 
+DROP TABLE  Position
+
+
+----------------------------
+
```
Comment (diyorbekaxmatov) [inline sql/DDL.sql:13]: What's the point of using the code in this line?

**Review by diyorbekaxmatov — APPROVED** [review]

File: sql/DDL.sql (hunk 4 above)
Comment (SharifovDeveloper) [reply sql/DDL.sql:13]: This code is used to separate lines

**Review by Nabiddinov — APPROVED** [review]

File: sql/DDL.sql [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+CREATE TABLE Document
```
[the remark this answers is not in the document]
Comment (SharifovDeveloper) [reply sql/DDL.sql:14]: Thank you

**Comment by Mirazyzz** [comment]
Looks good.

**Pull request merged**